src/mqttDeserialize.ts

```diff
@@ -87,8 +92,27 @@ export default class MqttParser {
       return { kind: 'fail', message };
     }
   }
+
+  /**
+   * Parse query string into regular javascript object.
+   *
+   * @param type Decoder for the deserialized object
+   * @param rawQuery e.g. "?userLat=61.123&userLng=24.55"
```

Comment:
I could be mistaken, but it seems to me that instead of userLat and userLng the query parameters should be just lat and lon, i.e.

?lat=61.123&lon=24.55

Reply:
`parseQuery` method on it's own should process all query strings as long as the `type` parameter is compatible with `rawQuery`.

But yeah the jsdoc is very misleading as we don't even use the shown query string format. I changed to jsdoc to this:

```
/**
   * Parse query string into regular javascript object.
   *
   * @param type Decoder for the deserialized object
   * @param queryString see https://en.wikipedia.org/wiki/Query_string
   */
```